broadway-builder-frontend/src/views/ViewAllTheaters.vue

```diff
@@ -8,8 +8,9 @@
           <!-- Clicking on the image goes to that theater -->
           <v-card>
             <v-img src="https://picsum.photos/510/300?random" height="200px" @click="goToProfile(theater)"></v-img>
-            <v-card-title primary-title>
-              <div @click="goToProfile(theater)">
+            <v-card-title primary-title @click="goToProfile(theater)">
+              <!-- <div @click="goToProfile(theater)"> -->
+              <div>
```

Comment:
The theater information dropdown no longer drops down. Instead when you click on the dropdown arrow it takes you to the theater profile. Lets bring the dropdown functionality back. I know that they all shouldn't dropdown at the same time but we can fix that bug in a separate PR. In the meantime lets keep the theater info dropdown functionality still there.